src/mechanics/integrands/TimeDependent.h

```diff
@@ -0,0 +1,56 @@
+#pragma once
+
+#include "mechanics/integrands/Base.h"
+#include "mechanics/integrands/Operations.h"
+#include "mechanics/nodes/DofVector.h"
+#include "mechanics/nodes/DofMatrix.h"
+
+namespace NuTo
+{
+namespace Integrand
+{
+namespace TimeDependent
+{
+struct Interface : Base
+{
+    virtual ~Interface() = default;
+    virtual DofVector<double> Gradient(const CellData&, const CellIpData&, double t, double deltaT) = 0;
```

Comment:
What about the suggestion here to not directly use the options t and delta_t, and instead pass a GradientOptionsObject that can be easily extended to accomodate other time dependent integrands.